akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -617,6 +659,20 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     }
   }
 
+  private def startAeronCounterLog(): Unit = {
+    import system.dispatcher
+    aeronCounterTask = system.scheduler.schedule(3.seconds, 5.seconds) {
```

Comment:
No problems running things on the scheduler of the system when the system startup is blocked?

Reply:
We also schedule in `startAeronErrorLog`. We could do these after we've waited for the transport to come up. Thoughts?

Reply:
If it works I think this is fine, was just surprised it would work.

Reply:
probably fine, might not work with dispatcher pool size of 1, but we have other things that will break with that

Reply:
could change the initial delay to 5.seconds to avoid that it is in done "at the same time" as the log polling

Reply:
Changed to 5